ui/lib/theme.ts

```diff
@@ -14,7 +14,7 @@ export const theme: DefaultTheme = {
   fontSizes: {
     huge: "48px",
     extraLarge: "32px",
-    large: "22px",
+    large: "20px",
```

Comment:
Really changing this globally?

Reply:
Bianca said higher up font sizes on the page should be either 14px or 20px - seemed like it'd be better to stay consistent than to put in a separate size for 22

Reply:
Yes, this matches our type scale https://www.figma.com/file/gnodK9xtoOahXBqa6ONsA5/WW-Design-System-%2F-Draft?node-id=4%3A5